### PR TITLE
 rtr: prefix interval mode symbols 

### PR DESCRIPTION
**Pull request by mroethke**
The intervals mode symbols do not have a prefix at all right now. This
is problematic because it could easily lead to name collisions when used
in other projects.
Furthermore are the {set,get}_interval_mode functions in the wrong place
they operate on rtr_sockets and should therefore be in rtr.c not in
rtr_mgr.c

While this is a breaking API change I argue that it does not really
matter because the interval mode code has never officially been
released.

### CODE REVIEW

File: rtrlib/rtr/rtr.h [hunk 1]
```diff
@@ -216,5 +216,20 @@ void rtr_stop(struct rtr_socket *rtr_socket);
  * @return !=NULL The rtr_socket_state as String.
  */
 const char *rtr_state_to_str(enum rtr_socket_state state);
+
+/**
+ * @brief Set the interval option to the desired one. It's either IGNORE_ANY,
+ * APPLY_ANY, DEFAULT_MIN_MAX or IGNORE_ON_FAILURE.
```
Comment (smlng) [inline rtrlib/rtr/rtr.h]: you may want to prefix this here, too?

**Comment by smlng** [comment]
otherwise looks good!

**Comment by colinbs** [comment]
Looks good to me!

**Comment by waehlisch** [comment]
@salsh you also don't use it in BGPStream, right?

**Comment by salsh** [comment]
@waehlisch you're right I don't use it

**Comment by mroethke** [comment]
done

**Pull request merged**